apps/drupal/src/Plugin/Preprocessor/Preprocessors/Node/Node.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+namespace Drupal\particle\Plugin\Preprocessor\Preprocessors\Node;
+
+use Drupal\particle\Plugin\Preprocessor\Preprocessors\Base;
+
+/**
+ * Provides a base node preprocessor.
+ */
+class Node extends Base {
+
+  /**
+   * {@inheritdoc}
+   */
+  public function init(array &$variables) {
+    parent::init($variables);
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function process(array &$variables) {
```

Comment:
While I think it's valuable to have this here as an example, I wonder if the above `init` function needs to be duplicated here, since that is unlikely to be customized per processor?

Reply:
Removed.